winpr/include/winpr/smartcard.h

```diff
@@ -115,6 +115,14 @@
 
 #endif
 
+/* missing definition with mingw */
+#ifndef SCARD_E_PIN_CACHE_EXPIRED
+#define SCARD_E_PIN_CACHE_EXPIRED (HRESULT)(0x80100032L)
```

Comment:
shouldn´t each of these be guarded by a `#ifndef`?

Reply:
Well I think it comes as a full set, but I can split it for more safety.